Item database update - Vicious Mind weapons

secretdataz opened the pull request:
This PR implements 17 weapons from Vicious Mind set.
kRO ItemInfo.lub and divine-pride is used as source.
http://www.divine-pride.net/database/item/1400
http://www.divine-pride.net/database/item/1450
http://www.divine-pride.net/database/item/1600
http://www.divine-pride.net/database/item/1800
http://www.divine-pride.net/database/item/1900
http://www.divine-pride.net/database/item/1996
http://www.divine-pride.net/database/item/2026
http://www.divine-pride.net/database/item/13128
http://www.divine-pride.net/database/item/13455
http://www.divine-pride.net/database/item/16041
http://www.divine-pride.net/database/item/18121
http://www.divine-pride.net/database/item/21016
http://www.divine-pride.net/database/item/28008
http://www.divine-pride.net/database/item/28107
http://www.divine-pride.net/database/item/28605
http://www.divine-pride.net/database/item/28706

Lemongrass3110 commented:
@rathena/rathena-script-db-developers

nanakiwurtz commented:
@secretdataz All okay, except the Job Bitmask, we usually use 8 digits just to prevent confusion.
And also for the 'bonus bUnbreakableWeapon,n;' we usually use 1 just to show that it's "On"/"Yes" (even though any value is the same).
After you follow up, this PR can be merged :smile_cat:

secretdataz commented:
@nanakiwurtz 
• Moved some miscategorized items
• Padded job bitmask with 0
• Use 1 in `bonus bUnbreakableWeapon,n`
• Fixed some typo in job bitmask
Thank you for your suggestions.
